packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1092,6 +1092,13 @@
       <protected>false</protected>
       <shortDescription>No related items</shortDescription>
     </labels>
+    <labels>
+      <fullName>quantic_NavigateToRecord</fullName>
```

Comment:
do we not want a translation for this label also or is that an english only label?

Reply:
translation added.